### ISSUE TITLE
WPA User Notification Banner

### ISSUE THREAD

**Issue by jeremyw24** [PlaceOS/user-interfaces#82]
Objective: Provide a banner that can display custom messages to users, users can dismiss the message and not see it again. 

Allow a custom message or notification to be entered and displayed in Workplace App, this can either be configured via Concierge Application where users of specific buildings could be targeted or alternatively implemented through Backoffice. 

This could be used to provide relevant information and important notices for example in the case of SaaS we may use it to notify users there will be an application upgrade scheduled at a specific time.

**Comment by MrYuion** [comment]
Add setting `app.general.banner` for this.
Data structure looks like this:
```javascript
{
    "id": "banner-1",
    "type": "info", // Can be `info`, `warn`, `error`
    "content": "Message to display to the user"
}
```

**Comment by jeremyw24** [comment]
@MrYuion can we have some ability to set this via concierge so end users, particularly on SaaS can set the message themselves?

**Comment by MrYuion** [comment]
Yeah, I'll add the ability to set it in the Backoffice extension.
